docs/design_decisions.md

```diff
@@ -0,0 +1,104 @@
+Functionele vraag is: ik wil in één vraag een persoon met adres hebben.
+Een groot deel van de vragen om persoonsgegevens is er direct behoefte aan de adresgegevens, zonder geïnteresseerd te zijn in de structuur in BAG (zoals naar nummeraanduiding, woonplaats, ligplaats, adresseerbaar object).
+Zie https://www.rvig.nl/brp/vraag-en-antwoord/mag-een-persoon-worden-ingeschreven-op-een-locatieomschrijving-in-de-gba-als-er-geen-authentiek-bag-adres-is
+Deze oplossing is ook een oplossing voor issue [22](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/issues/22) en [14](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/issues/14).
+
+## Verblijfplaats wordt gevuld met adresgegevens uit het GBA als er geen BAG adres beschikbaar is
+De verblijfplaats toont de adresgegevens zoals die in het GBA staan geregistreerd als er geen BAG nummeraanduiding en openbare ruimtenaam uit de BAG beschikbaar is.
+
+*Ratio*
+Vanuit het GBA hoeft er geen verbinding te zijn naar het BAG om de juiste adresgegevens op te halen.
+Er is niet altijd een BAG-adres.
+Zie https://www.rvig.nl/brp/vraag-en-antwoord/mag-een-persoon-worden-ingeschreven-op-een-locatieomschrijving-in-de-gba-als-er-geen-authentiek-bag-adres-is
+
+## Geboortedatum wordt gegevensgroep met dag, maand en jaar
+Ten behoeve van het ondersteunen van onvolledige datums, wordt de geboortedatum in de response een gegevensgroep "geboorte" met 4 properties:
+- geboortejaar: date-fullyear
+- geboortemaand: date-month
+- geboortedag: date-mday
+- geboortedatum: date
+
+Deze velden zijn gedefinieerd in ISO8601-typen.
+Als er een volledige geboortedatum is, worden alle 4 velden ingevuld, anders alleen de bekende delen.
+
+*Ratio*
+In de API moet het mogelijk zijn een persoon met gedeeltelijk onbekende geboortedatum op te nemen, zie issue [6](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/issues/6).
+De gekozen oplossing is het eenvoudigst te implementeren.
+
+## gemeenteVanInschrijving is een property met gemeentecode
+
+*Ratio*
+Gemeente is een objecttype (en daarmee waarschijnlijk een resource van de BAG). Daarom zou je verwachten dat gemeenteVanInschrijving een relatie is. In het model is het echter een attribuut met de gemeentecode.
+We houden dit een attribuut met gemeentecode, omdat dit gegeven wordt niet actief bijgehouden.
+
+## Uitgangspunt voor modellering is de basisregistratie
+Voor persoonsgegevens gebruiken we LO GBA 3.10.
+
+*Ratio*
+Het gaat om bevragen bij de bron. De bron voor persoonsgegevens is het GBA. Daarom moet het logisch ontwerp van de GBA worden gebruikt en er geen afwijkende RSGB-modellering zijn van persoonsgegevens.
+
+## Opvragen van historie kan via specifieke endpoints
+Opvragen van historie wordt gedefinieerd met aparte endpoints. Als een provider ervoor kiest om deze niet te implementeren wordt er een 501 geretourneerd. De historische bevraging krijgen dus resourcenaam waaruit blijkt dart het om een historische resource gaat.
```

Comment:
Nu dit gesplitst is, is dit volgens mij niet meer relevant

Reply:
Misschien niet in de design decisions, maar dit stukje tekst zou naar de readme of naar de description van de API kunnen. Ik maak een issue aan en laat tot die tijd de tekst hier staan.

Reply:
Misschien niet in de design decisions, maar dit stukje tekst zou naar de readme of naar de description van de API kunnen. Ik maak een issue aan en laat tot die tijd de tekst hier staan.